client/web/compose/src/themes/corteza-base/custom.scss

```diff
@@ -37,3 +37,7 @@
     }
   }
 }
+
+.gap-2 {
```

Comment:
Please dont add this here, its already defined globally with the new theme styles.
`/server/assets/src/scss/main`